colors.py

```diff
@@ -49,3 +49,37 @@ def nearest_match(self):
                     min = abs(item[1] - color_int)
                     color = item[0]
             return "Nearest Match: " + color
+    def RGB_cartesian_match(self):
+        #eucledian
+        color = None
+        min = 16777215  # FFFFFF
+        for item in colors[self.colset].values():
+            colorCompareHex = "0x{:06x}".format(item[1])
+            if self.eucleanCalculate(self.value, colorCompareHex) < min:
+                min = self.eucleanCalculate(self.value, colorCompareHex)
+                color = item[0]
+        return "Nearest Match: " + color
+	
+    def eucleanCalculate(self, C1, C2):
```

Comment:
Function and variable names should be lowercase, with words separated by underscores as necessary to improve readability as described in PEP8.